LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Amazon Web Services
```

Comment:
```suggestion
Copyright (c) 2020 Amazon Web Services
```

Reply:
TY for catching that, i just grabbed a file on my hard drive and missed that.